newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -109,6 +110,7 @@ export type PreferencesValues = {|
   useGDJSDevelopmentWatcher: boolean,
   eventsSheetUseAssignmentOperators: boolean,
   showEffectParameterNames: boolean,
+  lastUsedPath: Object,
```

Comment:
Verify this typing

Reply:
@4ian Verify here, or in the code as a comment? Basically, it is storing the object of all projects opened by the used with each kind of resource's path.

Reply:
`Object` means `any` in Flow, which is a weak typing not really helping to catch bugs.
See how to mark an object as a map here: https://flow.org/en/docs/types/objects/#toc-objects-as-maps

Reply:
Oh! I will update it based on that.